easybuild/easyconfigs/p/PyCharm/PyCharm-2019.3.1.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'Tarball'
+
+name = 'PyCharm'
+version = "2019.3.1"
+
+homepage = 'https://www.jetbrains.com/pycharm/'
+description = """PyCharm Community Edition: Python IDE for Professional Developers"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://download-cf.jetbrains.com/python']
+sources = ['pycharm-community-%s.tar.gz' % version]
+checksums = []
```

Comment:
@shahzebsiddiqui Please include a SHA256 checksum, and use `%(version)s`:

```python
sources = ['pycharm-community-%(version)s.tar.gz']

Reply:
i made the changes. Hope all is good now.